examples/storybook/src/stories/Hits.stories.ts

```diff
@@ -2,6 +2,7 @@ import { storiesOf } from '@storybook/angular';
 import { action } from '@storybook/addon-actions';
 import { wrapWithHits } from '../wrap-with-hits';
 import meta from '../meta';
+import { Hit } from 'instantsearch.js/es/types';
```

Comment:
the others are imported from Angular InstantSearch, this one from InstantSearch.js, does that have a reason?

Reply:
I'm just trying to use the one from InstantSearch when it's compatible, and fallback on the one in Angular IS only when I'm unable to get to work. 
Regarding specifically the `Hit` widget, the internal one is already mapped to InstantSearch's. If the goal is to reference IS's types directly, this is going a better change toward that direction imho.